test/volume-parse-test.js

```diff
@@ -0,0 +1,24 @@
+/* global describe it */
+const expect = require('chai').expect
+const { parseVolume } = require('../lib/volume-parser')
+
+describe('volume parsing', () => {
+  it('v. 36-37 (Nov. 1965-Oct. 1967)', () => {
+    expect(parseVolume('v. 36-37 (Nov. 1965-Oct. 1967)')).to.deep.equal([36, 37])
+  })
+  it('vol. 67 (May-Oct. 1996)', () => {
+    expect(parseVolume('vol. 67 (May-Oct. 1996')).to.deep.equal([67, 67])
+  })
+  it('volume 10, no. 1 - 4 (win. - aut. 1976) inde', () => {
+    expect(parseVolume('volume 10, no. 1 - 4 (win. - aut. 1976) inde')).to.deep.equal([10, 10])
+  })
+  it('vol 93, no. 3 (autumn 2013)', () => {
+    expect(parseVolume('vol 93, no. 3 (autumn 2013)')).to.deep.equal([93, 93])
+  })
+  it('v. 6-7 no. 2, 5-v. 8 no. 1 (Oct. 1961-Sept./Oct. 1962, May-June/July 1963)', () => {
+    expect(parseVolume('v. 6-7 no. 2, 5-v. 8 no. 1 (Oct. 1961-Sept./Oct. 1962, May-June/July 1963)')).to.deep.equal([[6, 7], [8, 8]])
+  })
+  it('v. 6-7 no. 2, 5-v. 8-10 no. 1 (Oct. 1961-Sept./Oct. 1962, May-June/July 1963)', () => {
+    expect(parseVolume('v. 6-7 no. 2, 5-v. 8-10 no. 1 (Oct. 1961-Sept./Oct. 1962, May-June/July 1963)')).to.deep.equal([[6, 7], [8, 10]])
+  })
+})
```

Comment:
Almost all of these will always fail in travis because the environment in travis isn't configured to connect to the dateparser lambda. So can we tag these as local only and exclude them from the standard test run? I think we can do that in mocha by:
 1. Add, for example, "@local-only" to the end of the `it` description
 2. Add ` --grep '@local-only' --invert` to the `mocha` call in the registered test script in `package.json`